docker/app/Dockerfile

```diff
@@ -38,24 +38,22 @@ RUN npm run build:${NPM_BUILD_SUFFIX}
 # COMPOSER-BUILDER
 # download composer app dependencies
 # git - needed for composer install
-FROM sillsdev/web-languageforge:base-php-7.4 AS composer-builder
+FROM sillsdev/web-languageforge:base-php AS composer-builder
 ENV COMPOSER_ALLOW_SUPERUSER=1
-RUN apt-get update && apt-get install -y git && rm -rf /var/lib/apt/lists/* \
-    && install-php-extensions @composer
 WORKDIR /composer
 COPY src/composer.json src/composer.lock /composer/
 RUN composer install
 
 # PRODUCTION IMAGE
-FROM sillsdev/web-languageforge:base-php-7.4 AS production-app
-RUN rm /usr/local/bin/install-php-extensions
-RUN apt-get remove -y gnupg2
+FROM sillsdev/web-languageforge:base-php AS production-app
+RUN rm /usr/local/bin/install-php-extensions /usr/local/bin/composer
```

Comment:
we should remove composer from the production image - should have done this earlier but didn't realize it